server/src/test/integration/usersEndpointTest.js

```diff
@@ -2,30 +2,45 @@ const app = require('../../server');
 const request = require('supertest');
 const expect = require('chai').expect;
 const dbAdapter = require('../../db/adapter');
+const auth = require('../../../config').auth;
+const axios = require('axios');
 
 describe(`'users' route - api test`, function() {
-  // before the tests, prime the database with test tables/data
+  let token;
+
   before(async function() {
+    // prime the database with test tables/data
     const tables = await dbAdapter.r.tableList();
-
     if (tables.includes('users')) {
       await dbAdapter.r.tableDrop('users');
     }
-
     await dbAdapter.r.tableCreate('users');
-    return Promise.resolve();
+
+    // get a temporary bearer token for testing
+    const options = {
```

Comment:
Maybe explore some kind of pre-test script or some config where you can inject globally used variables like this mock token